lib/routes/update-snapshot.js

```diff
@@ -0,0 +1,43 @@
+const e = require('express');
+const snapShot = require('snap-shot-core');
+
+/** @type {e.RequestHandler} */
+const updateSnapshot = (request, response) => {
+  const {
+    body,
+    params: { modulename = 'default-module', testname, snapname }
+  } = request;
+  let errored = false;
+
+  const opts = {
+    show: Boolean(process.env.SNAPSHOT_SHOW),
```

Comment:
Should we extract this to another module (and fixup the sanitization)? Perhaps in a follow up PR though...

Reply:
addressed here 876d75f